src/wp-includes/blocks.php

```diff
@@ -1250,14 +1250,18 @@ function unregister_block_style( $block_name, $block_style_name ) {
  * @since 5.8.0
  *
  * @param WP_Block_Type $block_type    Block type to check for support.
- * @param array         $feature       Path to a specific feature to check support for.
+ * @param string|array  $feature       Path to a specific feature to check support for.
  * @param mixed         $default_value Optional. Fallback value for feature support. Default false.
  * @return bool Whether the feature is supported.
  */
 function block_has_support( $block_type, $feature, $default_value = false ) {
 	$block_support = $default_value;
 	if ( $block_type && property_exists( $block_type, 'supports' ) ) {
-		$block_support = _wp_array_get( $block_type->supports, $feature, $default_value );
+		if ( is_array( $feature ) ) {
+			$block_support = _wp_array_get( $block_type->supports, $feature, $default_value );
+		} else {
+			$block_support = isset( $block_type->supports[ $feature ] ) ? $block_type->supports[ $feature ] : $default_value;
+		}
```

Comment:
```suggestion
		} elseif ( isset( $block_type->supports[ $feature ] ) ) {
			$block_support = $block_type->supports[ $feature ];
		}
```

The default value already set at starting of the function so i wil use elseif.

Reply:
We should also add here another condition based on @swissspidy's feedback in https://core.trac.wordpress.org/ticket/58532#comment:4.